### PR TITLE
Minor updates to README.md, including added info about how to run utest.

### PR DESCRIPTION
**Pull request by david-broman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ MCore consists of two parts:
 
 ## MExpr
 
-One design objective of MExpr is to make the concrete syntax very close to the abstract syntax of the language. That is, no syntactic sugar is introduced to the concrete MCore syntax. The MExpr language is not intended to be a general purpose programming language, but a core language to which other languages translate into.
+One design objective of MExpr is to make the concrete syntax very close to the abstract syntax of the language. That is, no syntactic sugar is introduced to the concrete MCore syntax. The MExpr language is not intended to be a general purpose programming language, but as a core language to which other languages translate into.
```
Comment (dlunde) [inline README.md]: "but a core language" sounds better than "but as a core language"

File: README.md [hunk 2]
```diff
@@ -92,7 +92,14 @@ When writing MCore programs, it is typically done by writing explicit unit tests
 utest addi 1 2 with 3 in
 ()
 ```
-checkes that the addition of `1` and `2` is in fact `3`. Typically when you develop MCore programs, you do not use the `print` function. Instead, you write unit tests directly, and then leave the units tests as is directly after your function. By doing so, you test your code, write regression tests, and document the informal semantics of your program directly. We strongly encourage you to develop your MCore programs this way.
+checkes that the addition of `1` and `2` is in fact `3`. To run the tests in an `.mc` file, run the `boot` command with argument `test`:
```
Comment (dlunde) [inline README.md]: "checkes" -> "checks"

**Review by dlunde — COMMENTED** [review]
Looked through changes.

File: README.md (hunk 1 above)
Comment (david-broman) [reply README.md]: Well, I disagree. Anyway, I have now reformulated the sentence by breaking it up into two sentences instead.

File: README.md (hunk 2 above)
Comment (david-broman) [reply README.md]: Fixed, thanks!

**Pull request merged**